compose/compose-file/index.md

```diff
@@ -348,33 +348,12 @@ Simple example:
 > for a service to be ready, see [Controlling startup order](/compose/startup-order.md)
 > for more on this problem and strategies for solving it.
 
-A healthcheck indicates you want a dependency to wait
-for another container to be "healthy" (i.e. its healthcheck advertises a
-successful state) before starting.
+> **NOTE:** version 3 no longer supports the `condition` form of `depends_on`
```

Comment:
I disagree. That section is for fields that have been removed entirely, because there is no where else to document them. In this case the field still exists, but it only supports the "short form".  I think users would expect to see this documented in the field section.